packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -446,10 +446,9 @@
                             data-on-click="$ctrl.gotToNetworks()"
                         >
                         </oui-action-menu-item>
-                        <oui-action-menu-item
-                            text="{{:: 'pci_projects_project_instances_instance_private_networks_action_attach_label' | translate }}"
-                            data-on-click="$ctrl.attachPrivateNetwork()"
-                        >
+                        <oui-action-menu-item text="{{:: 'pci_projects_project_instances_instance_private_networks_action_attach_label' | translate }}"
```

Comment:
I think `text` attribute is deprecated. A nested `span` would be better

Reply:
This is done. Thanks